### DISCUSSION TITLE
Allure report without Jetty, without having to serve the HTML files

### DISCUSSION THREAD

**Discussion by pandoras-toolbox** [allure-framework/allure2#1981] [Questions & Support] [answered]
We want to archive the Allure reports in a MINIO Object Store tool.

The Allure reports require Jetty to serve them. That is my problem. If I store the Allure reports in MINIO Object Store and try to open the report, then a just blank page is displayed, which is not what is required.

Is it possible to create a standalone Allure report which does not require Jetty to serve them, meaning that not a blank page will be displayed in the browser when opening the HTML report directly without Jetty?

Currently I try to replace the Cucumber reports with Allure reports (and Cucumber with JUnit 5). The Cucumber reports do not require anything to serve them, so it works fine to host them in a MINIO Object Store.

In order to get approval for my suggestion to replace Cucumber reports with Allure reports I need to solve storing Allure reports in MINIO Object Store in such a way that it does not require Jetty to serve them. Using Allure TestOps instead is currently no option unfortunately because for now they do not want to change how reports are stored.

What would you suggest?

**Comment by baev** [comment, marked as answer]
Allure doesn't require the Jetty to be opened. The reason Jetty is used to serve the report is that Allure uses Ajax requests to fetch the report data. And when the report is served from the local filesystem, such requests are forbidden by Webkit security settings. 

Although you can store the reports in any storage that supports path-style access to the files

**Comment by pandoras-toolbox** [comment]
Thank you for the quick reply. After I created the issue I had the idea to directly upload an Allure report folder to MINIO and to check if will be a blank page like locally. But it works, so it is good.
